library/camellia.c

```diff
@@ -560,6 +578,12 @@ int mbedtls_camellia_crypt_cbc( mbedtls_camellia_context *ctx,
 {
     int i;
     unsigned char temp[16];
+    CAMELLIA_VALIDATE_RET( ctx != NULL );
+    CAMELLIA_VALIDATE_RET( mode == MBEDTLS_CAMELLIA_ENCRYPT ||
+                           mode == MBEDTLS_CAMELLIA_DECRYPT );
+    CAMELLIA_VALIDATE_RET( iv != NULL );
+    CAMELLIA_VALIDATE_RET( length == 0 || input  != NULL );
```

Comment:
As I've said in other PR's, I'd like a test for when `length == 0 && input  == NULL`.

Reply:
As in the other PRs, that's a positive test and hence deliberately omitted here.